src/NUnitFramework/framework/Constraints/DictionaryContainsKeyConstraint.cs

```diff
@@ -86,13 +84,15 @@ protected override bool Matches(IEnumerable actual)
         /// </summary>
         /// <param name="comparison">The comparison function to use.</param>
         /// <returns>Self.</returns>
-        public DictionaryContainsKeyConstraint Using<TCollectionType, TMemberType>(Func<TCollectionType, TMemberType, bool> comparison)
+        public DictionaryContainsKeyConstraint Using<TCollectionType, TMemberType>(
+            Func<TCollectionType, TMemberType, bool> comparison)
         {
             // reverse the order of the arguments to match expectations of PredicateEqualityComparer
-            Func<TMemberType, TCollectionType, bool> invertedComparison = (actual, expected) => comparison.Invoke(expected, actual);
+            Func<TMemberType, TCollectionType, bool> invertedComparison =
+                (actual, expected) => comparison.Invoke(expected, actual);
```

Comment:
We typically ask people to not change unrelated formatting. I think these two changes may have been done automatically by your IDE.
If you're comfortable doing it, you can edit them back to the way they were before, amend your current commit, and force push to your `RequireActual` branch and it'll automatically update here. If you're not (I see you're a first-time GitHib contributor) then don't worry about it. It's easier to catch this stuff by viewing the diff on your machine before pushing to your GitHub fork.